### PR TITLE
make windowed size take into account window border and title height

### PR DESCRIPTION
**Pull request by fr500**
## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises

## Description

Fixes this partially: https://github.com/libretro/RetroArch/issues/7670
Basically now the variables control the windowed viewport size instead of the whole rectangle

### CODE REVIEW

**Comment by fr500** [comment]
Added a commit, window should no longer grow with menu bar enabled
@waitingmoon
